### ISSUE TITLE
config value for disabling "recent repositories" prompt on startup

### ISSUE THREAD

**Issue by donjo9** [jesseduffield/lazygit#3129]
**Is your feature request related to a problem? Please describe.**
When ever i open Lazygit in a non git dir, it shows the recent repo list, even though the last repo opened is showing.

**Describe the solution you'd like**
I would like a config value for disabling that prompt.

**Describe alternatives you've considered**
Alternative make it so when ever ```notARepository ``` is set to ```skip``` it skips the "recent repository" aswell

**Additional context**
![billede](https://github.com/jesseduffield/lazygit/assets/6575408/683fd9b0-0211-4c4d-bc46-f516a7f193a8)

![billede](https://github.com/jesseduffield/lazygit/assets/6575408/ba836681-a200-49ca-9308-5c64ec04b3ba)

**Comment by RC-TonyDemetriou** [comment]
I see this is fairly old & the PR for it has been closed - is this still something that could happen?

It's fairly minor to press esc each time, but when we've set config to default to the last-opened-repo it'd be convenient to only see this menu if lazygit can't open or doesn't know which repo was last used :)

**Comment by stefanhaller** [comment]
Yes, it definitely could happen; it's just not very high on my priority list. That PR wasn't quite the right solution if I remember right; this needs a bit more design, and I don't want to spend time on that right now.

**Comment by isdba** [comment]
I'd love that change too and it would improve my life a lot by reducing the amount of ESC keystrokes. I often launch lazygit inside neovim where I have serveral projects open in parallel but mostly need the last repo. It is also what I'd expect from the description of the "skip" option for notARepository which is "open most recent repo". The implicit repo selector is unexpected from that description (at least for me).
